### PR TITLE
News screen: Align UI implementation with design

### PR DESCRIPTION
**Pull request by ice-atlas**
- change pull-to-refresh behaviour;
- add "no more items" view;
- add loading (more) state;
- use bottom sheet view for list of news;
- implement transition animation for featured news article header.

### CODE REVIEW

File: src/constants/colors.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ export const COLORS = {
   white02opacity: 'rgba(255,255,255, .2)',
   black01opacity: 'rgba(0,0,0,.1)',
   primary: '#073e91',
+  primaryDarkTransparent: '#0D265E00',
```
Comment (ice-orion) [inline src/constants/colors.ts]: do we need it?

File: src/screens/News/components/FeaturedNewsArticle/hooks/useLayoutAnimation.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        },
+      ],
+    };
+  });
+
+  return {
+    contentStyle,
+    titleStyle,
+    valuesContainerStyle,
+
```
Comment (ice-hector) [inline src/screens/News/components/FeaturedNewsArticle/hooks/useLayoutAnimation.ts]: remove unneeded space

File: src/constants/colors.ts (hunk 1 above)
Comment (ice-atlas) [reply src/constants/colors.ts]: removed

File: src/screens/News/components/FeaturedNewsArticle/hooks/useLayoutAnimation.ts (hunk 2 above)
Comment (ice-atlas) [reply src/screens/News/components/FeaturedNewsArticle/hooks/useLayoutAnimation.ts]: ok

File: src/assets/svg/NoMoreNewsIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  width = rem(33),
+  height = rem(33),
+  color = COLORS.periwinkleGray,
+  ...props
+}: SvgProps) => (
+  <Svg
+    width={width}
+    height={height}
+    fill={'none'}
+    viewBox={'0 0 35 36'}
```
Comment (ice-hector) [inline src/assets/svg/NoMoreNewsIcon.tsx:18]: Is it fine that default viewBox size vary from default width/height that you set?

File: src/screens/News/hooks/useOnRefresh.ts [hunk 4]
```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: ice License 1.0
```
Comment (ice-hector) [inline src/screens/News/hooks/useOnRefresh.ts:1]: Recheck all places where you've duplicated licences and remove old one `// SPDX-License-Identifier: BUSL-1.1`

File: src/assets/svg/NoMoreNewsIcon.tsx (hunk 3 above)
Comment (ice-atlas) [reply src/assets/svg/NoMoreNewsIcon.tsx:18]: yes, it is exactly by design. SVG will fit in the specified sizes

File: src/screens/News/hooks/useOnRefresh.ts (hunk 4 above)
Comment (ice-atlas) [reply src/screens/News/hooks/useOnRefresh.ts:1]: good catch

**Review by ice-hector — DISMISSED** [review]

**Review by ice-orion — DISMISSED** [review]

**Review by ice-hector — APPROVED** [review]

**Review by ice-orion — APPROVED** [review]

**Review by ice-hades — APPROVED** [review]

**Pull request merged**